Pipfile

```diff
@@ -4,28 +4,28 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
+moto = "*"
+pre-commit = "*"
 pytest = "*"
 pytest-cov = "*"
-pre-commit = "*"
-moto = "*"
 
 [packages]
+aenum = "*"
 boto3 = "*"
 click = "*"
 geojson = "*"
-parallelpipe = "*"
+google-cloud-storage = "*"
+parallelpipe = {editable = true,git = "https://github.com/wri/parallelpipe.git",ref = "fix_for_python_3.8"}
```

Comment:
I'm curious, does it seems like this is actually going to continue to be fixed/supported for future Python versions, since we're trying to update even further?

Reply:
Actually just noticed we're upgrading to 3.10. Does this work with 3.10?

Reply:
So I WAS going to say that the upstream project hasn't been updated in years. If you didn't notice, this is a fork I created 7 days ago because it DIDN'T work with 3.10. BUT it seems that my forking it got the attention of the author, because 5 days ago they fixed the issues and made a new release! I had always meant to make a PR to the original project, but I guess I don't have to now (and indeed can abandon this fork). Best possible outcome! :)